docs/authoring-components.md

```diff
@@ -231,14 +232,9 @@ methods. We can now take these fake methods and transform them into an adapter i
 | SOMEHOW_ADD_CLASS(className: string) | addClass(className: string) |
 | SOMEHOW_REMOVE_CLASS(className: string) | removeClass(className: string) |
 | SOMEHOW_UPDATE_TOGGLE_COLOR_TEXT_CONTENT(textContent: string) | setToggleColorTextContent(textContent: string) |
-| SOMEHOW_REGISTER_INTERACTION_HANDLER(type: string, handler: EventListener) | registerInteractionHandler(type: string, handler: EventListener) |
-| SOMEHOW_DEREGISTER_INTERACTION_HANDLER(type: string, handler: EventListener) | deregisterInteractionHandler(type: string, handler: EventListener) |
 
-> Note: It is a convention in our code base to use the terms `registerInteractionHandler` and
-`deregisterInteractionHandler` as adapter methods for adding/removing generic event listeners. We do
-this because we feel it more semantic in that most components are only interested in interactivity.
-However, feel free to call these methods `{add,remove}EventListener` or anything else you would
-like.
+> Note: We have changed our convention of using `registerInteractionHandler` and
+`deregisterInteractionHandler` as adapter methods for adding/removing generic event listeners. Instead please manage events in the component layer. This is because event systems can change from framework/environment (ie. React Synthetic Events).
```

Comment:
Change this:

```markdown
Instead please manage events in the component layer.
This is because event systems can change from framework/environment
```

to this:

```markdown
Event handling varies significantly across frameworks
(e.g., [React Synthetic Events](https://reactjs.org/docs/events.html)),
so we recommend managing events in the component layer.
```